README.md

```diff
@@ -126,25 +126,15 @@ pip install -e .
 
 ### Linux
 
-First, you need to install protobuf.
-
-Ubuntu users: the quickest way to install protobuf is to run
+First, you need to install protobuf. The minimum Protobuf version required by ONNX is 3.0.0.
 
+Ubuntu 22.04 (and newer) users may choose to install protobuf via
 ```bash
 apt-get install python3-pip python3-dev libprotobuf-dev protobuf-compiler
 ```
+In this case, it is recommended to add `-DONNX_USE_PROTOBUF_SHARED_LIBS=ON` to CMAKE_ARGS in the ONNX build step.
 
-Then you can build ONNX as:
-```
-export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
-git clone --recursive https://github.com/onnx/onnx.git
-cd onnx
-# prefer lite proto
-set CMAKE_ARGS=-DONNX_USE_LITE_PROTO=ON
-pip install -e .
-```
-
-Otherwise, you may need to install it from source. You can use the following commands to do it:
+A more general way is to build and install it from source. You can use the following commands to do it:
```

Comment:
Frankly speaking, I think the old build instructions are poorly organized (so this PR is very necessary). Most of the instructions are about how to install the third-party protobuf instead of onnx itself. How about removing the line 139-150 (because protobuf can be installed by apt in debian/ubuntu), and [folding](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) the line 152-163?